internetarchive/cli/ia_upload.py

```diff
@@ -176,6 +181,10 @@ def main(argv, session):
                   'Expect 503 SlowDown errors.'.format(args['<identifier>']),
                   file=sys.stderr)
             sys.exit(1)
+        elif get_item('{0}'.format(args['<identifier>'])).item_size >= 1099511627776:
```

Comment:
could we use a global, in capitals, at the top of the file here instead of this magic number? something like:

```
IA_MAX_SIZE = 1099511627776
```

or even better, what the actual value *means*:

```
# one tibibyte
IA_MAX_SIZE = 2^40
```

Reply:
I agree with @anarcat. Also, we already initialize the item on https://github.com/jjjake/internetarchive/blob/master/internetarchive/cli/ia_upload.py#L183, this check should happen after that so we only have to do it once.